apps/nextjs/src/app/dashboard/@breadcrumbs/default.tsx

```
@@ -0,0 +1,51 @@
+}
+
+const defaultItems = [{ label: "Dashboard", href: "/dashboard" }];
+
+// TODO: Update Breadcrumbs to actually update when the route changes
+export default function Breadcrumbs({
+  items = defaultItems,
+}: BreadcrumbsProps) {
+  return (
+    <Breadcrumb aria-label="Navigation breadcrumbs">
+      <BreadcrumbList>
+        <BreadcrumbItem></BreadcrumbItem>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)